remove-webhook/README.md

```diff
@@ -17,6 +17,7 @@ read -s RANCHER_TOKEN && export RANCHER_TOKEN
 read -s RANCHER_URL && export RANCHER_URL
 bash remove-webhook.sh
 ```
+For rancher setups using self-signed certificates, you can specify `--insecure-skip-tls-verify` to force the script to ignore tls cerificate verification. Note that this option is insecure, and should be avoided for production setups.
```

Comment:
Nit: because this might be customer-facing in the future, I'd say `Rancher` and `TLS`. Also, typo in "certificate" next to TLS.

Reply:
Changes made.